docs/sources/configure-client/_index.md

```diff
@@ -57,16 +60,27 @@ Here's how to use Pyroscope SDKs:
 By using the Pyroscope SDKs, you have the flexibility to customize the profiling process according to your application's specific requirements.
 You can selectively profile specific sections of code or send profiles at different intervals, depending on your needs.
 
-## Choose Grafana Alloy or Pyroscope SDK to send profiles
+## About instrumentation with Pyroscope SDKs through Alloy
 
-You can use Grafana Alloy for auto-instrumentation or the Pyroscope instrumentation SDKs.
+Pyroscope SDKs can be configured to send profiles to Grafana Alloy first, which then forwards them to the Pyroscope server. This method combines the flexibility of SDK instrumentation with Alloy's infrastructure benefits.
+
+Here's how it works:
+1. Your application is instrumented with Pyroscope SDKs
+2. Instead of sending profiles directly to Pyroscope, the SDK sends them to Alloy's `pyroscope.receive_http` component
+3. Alloy processes and forwards the profiles to the Pyroscope server
+
+By sending profiles through Alloy, you benefit from lower latency as profiles are sent to a local Alloy instance instead of directly over the internet to Grafana Cloud. Your application code remains focused on instrumentation while infrastructure concerns like authentication and routing are handled by Alloy's configuration. This separation allows for centralized management of metadata, where you can enrich profiles with infrastructure labels such as Kubernetes metadata or environment tags without modifying application code.
```

Comment:
This is a minor suggestion to follow best practices of having one sentence per line. 
```suggestion
By sending profiles through Alloy, you benefit from lower latency as profiles are sent to a local Alloy instance instead of directly over the internet to Grafana Cloud.
Your application code remains focused on instrumentation while infrastructure concerns like authentication and routing are handled by Alloy's configuration.
This separation allows for centralized management of metadata, where you can enrich profiles with infrastructure labels such as Kubernetes metadata or environment tags without modifying application code.
```